src/System.Drawing.Common/src/System/Drawing/Printing/LibcupsNative.cs

```diff
@@ -9,10 +9,10 @@ namespace System.Drawing.Printing
 {
     internal static class LibcupsNative
```

Comment:
You should add a static constructor here that calls `PlatformInitialize` to set up the `DllImportResolver`. Without that, it's possible for something to call a P/Invoke here without that set up yet.